### ISSUE TITLE
Merge animation and mesh

### ISSUE THREAD

**Issue by EarthKiii** [Vorono4ka/scw2dae#8]
I need help to merge mesh and armature ! i used scw2dae
Thanks !

**Comment by danila-schelkov** [comment]
It should automatically merge if scw files have a similar name

Stop. I misunderstood you.

Mesh and armature shound merge if they are in the same file

**Comment by EarthKiii** [comment]
But i got 2 different files in dae folder, because i put _geo and _idle in the scw folder

**Comment by danila-schelkov** [comment]
It's right, u got geo file and animation file with geo

**Comment by EarthKiii** [comment]
But my animation file is named _idle

**Comment by EarthKiii** [comment]
and it has no mesh

**Comment by danila-schelkov** [comment]
Do you have the latest version of the module and script? Tell the filenames

**Comment by EarthKiii** [comment]
![image](https://user-images.githubusercontent.com/67222685/112337764-7e906800-8cbe-11eb-8d63-95833214981b.png)
is this what you need ? Which files ?

**Comment by danila-schelkov** [comment]
I mean scw filenames

**Comment by EarthKiii** [comment]
barkeep_geo.scw and barkeep_idle.scw

**Comment by danila-schelkov** [comment]
Can u give dae files that u got from script

**Comment by Voidoffi** [comment]
[Uploading dae.zip…]()

**Comment by EarthKiii** [comment]
[dae.zip](https://github.com/Vorono4ka/scw-tool/files/6198659/dae.zip)

**Comment by EarthKiii** [comment]
any update ?

**Comment by Ygor888** [comment]
Hello! I have 1 question. Why when I open main.py and choose the second tool, the folders "dae" and "scw" are not created?

**Comment by danila-schelkov** [comment]
Hi! Possible, you've forgotten to install requirements. Use `pip install -r requirements` in console in script folder.

**Comment by Ygor888** [comment]
I still don't understand how to apply your instructions. Here is a video where it is shown that after opening main.py , folders "dae" and "scw" are not created. Please send video instructions on how to solve this problem. (Also update the Youtube video tutorial on converting scw2dae).
https://user-images.githubusercontent.com/97162219/156890108-7c2a420c-779b-4aab-82be-38ac7b0296da.mp4

**Comment by danila-schelkov** [comment]
There is only Winrar window is captured. But i got u, so i will update the video tutorial

**Comment by Ygor888** [comment]
Hello! You have not updated the video instructions for importing scw2dae

**Comment by Ygor888** [comment]
And hi again! Here I show in detail how your tool works for me. First I open the most recent version of the tool. And I open it main.py . I choose the second tool (scw2dae) and nothing happens (folders are not created). Then I open the very first version of the tool. After, I open assets_downloader.py , and after that, this window opened and also closed quickly. Then I open it reader.py , and here is the same situation as with assets_downloader.py . After, I finally opened scw2dae.py, and here is the same situation as I opened reader.py .
Maybe I'm doing something wrong here, but I followed all the instructions you provided.

**Comment by Ygor888** [comment]
And hi again! Here I show in detail how your tool works for me. First I open the most recent version of the tool. And I open it main.py . I choose the second tool (scw2dae) and nothing happens (folders are not created). Then I open the very first version of the tool. Then, I open assets_downloader.py , and after that, this window opened and also closed quickly. Then I open it reader.py , and here is the same situation as with assets_downloader.py . After, I finally opened scw2dae.py, and here is the same situation as I opened reader.py .
Maybe I'm doing something wrong here, but I followed all the instructions you provided.

https://user-images.githubusercontent.com/97162219/159120716-2a555895-4c69-4078-9752-c5d206530d93.mp4
